doc/Overview/Localization/LocalizationOverview.md

```diff
@@ -1,46 +1,70 @@
 ---
 uid: Overview.Localization
 ---
+
 # Localization
-Uno.Extensions.Localization uses [Microsoft.Extensions.Localization](https://www.nuget.org/packages/Microsoft.Extensions.Localization) for any localization related work.
 
-For more documentation on localization, read the references listed at the bottom.
+It is often necessary to adapt an application to a specific subset of users within a market. **Localization** includes the type of actions developers take to modify both user interface elements and content to adhere to more languages or cultures. Specifically, text **translation** is done by applying alternate strings of text at runtime which accomodate a user's language preference.
 
-## Text localization
+Many apps store these pieces of text in dedicated resource files that the app parses and assigns as text throughout the application. `Uno.Extensions.Localization` provides a consistent way to resolve text of a specific culture or locale across platforms. This feature allows for modifying them to be applied upon app restart.
 
-Save locale specific resources in `.resw` files in folder corresponding to locale (eg en-US).
+It uses [Microsoft.Extensions.Localization](https://www.nuget.org/packages/Microsoft.Extensions.Localization) for any localization related work. For documentation on the broader process of localization, read the references listed at the bottom.
 
-An implementation of `IStringLocalizer` (`ResourceLoaderStringLocalizer`) is registered as service.
+## Set up localization
 
 ```csharp
-private IHost Host { get; }
-
-public App()
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLocalization()
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => 
+        {
+            host.UseLocalization()
+        });
+...
+```
+
+An implementation of `IStringLocalizer` (`ResourceLoaderStringLocalizer`) will be registered as a service. This service offers a consistent way to resolve localized strings. Behind the scenes, it will automatically use `ResourceManager` on Windows and `ResourceLoader` on other platforms.
+
+### Adding language specific resources
```

Comment:
You need to add any additional languages to the configuration file that lists supported languages